### DISCUSSION TITLE
코드블럭 관련 버그입니다.

### DISCUSSION THREAD

**Discussion by drawcodeboy** [pronist/hELLO#174] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: windows
- 브라우저: chrome
- 버전: [e.g. 22]
- 스킨 버전 3.6.5

**설명**
스킨을 적용하거나 기존의 포스팅을 수정함에 따라
코드블럭의 언어들이 모두 HTML로 바뀌게 됩니다.
(ex python -> HTML)
새 글을 작성할 때, 적용하는 언어들은 모두 정상적으로 적용됩니다.

**Comment by pronist** [comment, marked as answer]
@drawcodeboy 그 부분은 점검이 필요합니다. **티스토리 에디터** 자체에서 종종 그런 버그를 냅니다. 이미 작성된 코드 블록을 수정하는데, 디폴트 값이 묘하게 설정되는 경우죠. 저도 겪습니다. 이는 스킨의 버그가 아닙니다.  따라서 이 경구 면밀한 체크가 필요합니다. 이 이슈는 디스커션으로 옮깁니다.
